Jellyfin.Server/Middleware/ExceptionMiddleware.cs

```diff
@@ -85,6 +91,14 @@ public async Task Invoke(HttpContext context)
 
                 context.Response.StatusCode = GetStatusCode(ex);
                 context.Response.ContentType = MediaTypeNames.Text.Plain;
+
+                // Don't send exception unless the server is in a Development environment
+                if (!_hostEnvironment.IsDevelopment())
+                {
+                    await context.Response.WriteAsync("Error processing request.").ConfigureAwait(false);
+                    return;
+                }
+
                 var errorContent = NormalizeExceptionMessage(ex.Message);
```

Comment:
```suggestion
                var errorContent = _hostEnvironment.IsDevelopment()
                        ? NormalizeExceptionMessage(ex.Message)
                        : "Error processing request.";
```

this seems shorter and less code-duplicating